### PR TITLE
Katie enhancement 34 update copy

### LINKED ISSUE #34: Update Copy throughout Site

### PR DESCRIPTION
**Pull request by kcbreuil**
closes #34

### CODE REVIEW

**Comment by kcbreuil** [comment]
@asasmith Some of this is missing content from the client, but we should be good for the homepage and initiatives that will be shared with them tomorrow

**Comment by kcbreuil** [comment]
closes #34

File: src/pages/ideas.js [hunk 1]
```diff
@@ -9,39 +9,39 @@ import ListComponent from '../components/listComponent';
...
-                    '6 Cras iaculis, lectus a condimentum lacinia, risus ex varius est, vel fermentum magna enim sed eros.',
+                    '-Empirical: Can its influence be observed and measured?',
             },
             {
-                text: '7 A longer list item ',
+                text: '-Integrative: Does it intersect with more than one of Silverado’s three focus areas?',
             },
             {
-                text: '8 A longer list item ',
+                text: '-Scalable: Can its scope be broadened to address related policy challenges?',
```
Comment (asasmith) [inline src/pages/ideas.js:44]: lets remove the hyphen from these

Comment (kcbreuil) [reply src/pages/ideas.js]: @asasmith done!

**Review by asasmith — APPROVED** [review]

**Pull request merged**